addons/blocks2image/addon.json

```
@@ -1,11 +1,15 @@
 {
   "name": "Save blocks as image",
-  "description": "Right click the code area to export blocks as SVG/PNG images.",
+  "description": "Right-click on the code area or a stack of blocks to copy it to the clipboard or save it as a PNG or SVG file.",
   "tags": ["editor", "codeEditor"],
   "credits": [
     {
       "name": "summerscar",
       "link": "https://github.com/summerscar"
+    },
+    {
+      "name": "Jazza",
+      "link": "https://scratch.mit.edu/users/greeny--231"
     }
```

Can also add mxmou if he wants

```suggestion
    },
    {
      "name": "World_Languages"
    }
```